multimap/src/main/java/org/infinispan/multimap/impl/function/BaseFunction.java

```diff
@@ -0,0 +1,14 @@
+package org.infinispan.multimap.impl.function;
+
+import java.util.Collection;
+
+import org.infinispan.functional.EntryView;
+import org.infinispan.util.function.SerializableFunction;
+
+/**
+ * A base function for the multimap updates
+ *
+ * @author Katia Aresti, karesti@redhat.com
+ * @since 9.2
+ */
+abstract class BaseFunction<K, V, R> implements SerializableFunction<EntryView.ReadWriteEntryView<K, Collection<V>>, R> {}
```

Comment:
I created in class at first because I believed code will be shared, but indeed, looks more like that now, so I change it